tsdb/db.go

```diff
@@ -968,6 +974,8 @@ func (db *DB) run(ctx context.Context) {
 	defer close(db.donec)
 
 	backoff := time.Duration(0)
+	compactOffset := time.Duration(db.opts.CompactOffset) * time.Millisecond
```

Comment:
Can `db.opts.CompactOffset` be of type `time.Duration` to simplify this?

Reply:
Okk, I thought only built-in/basic types are "authorized" as DB options.